velox/vector/VectorStream.h

```diff
@@ -43,6 +43,15 @@ class VectorSerializer {
   /// Serialize all rows in a vector.
   void append(const RowVectorPtr& vector);
 
+  // Usage
+  // append(vector, ranges);
+  // size_t size = serializedSize();
+  // OutputStream* stream = allocateBuffer(size);
+  // flush(stream);
+  //
+  // Return OutputStream accurate required size
+  virtual size_t serializedSize() = 0;
```

Comment:
CompactRowVectorSerializer does not override this function, so there is a compilation issue.